support/python/mkdocstrings_handlers/cxx/__init__.py

```diff
@@ -5,336 +5,336 @@
 from __future__ import annotations
```

Comment:
Is this needed? I got an impression from the discussion on the previous PR that it was added but not used.

Reply:
Good spot:
- Indeed, I had initially added this import because it allowed us to use the operator '|' (e.g., `List[ElementTree.Element] | ElementTree.Element`.
- But then we change the code so that we were not using operator '|'.
- However, I hadn't removed it because, if I did it, I was getting a compilation error.
- I've fixed now that compilation error, by using `-> List[Definition]` instead of `-> list[Definition]` at `convert_params`.